docs/design/design_v1.9.md

```diff
@@ -1,4 +1,4 @@
-## Implementation design for kubeadm
+# Implementation design for kubeadm
```

Comment:
please don't include this change.
there rest seem fine.

also, please keep the commits squashed to 1.